docs/_sidebar.md

```diff
@@ -0,0 +1,17 @@
+* [Home](README.md)
+* [The Arena](arena.md)
+* [The Gooeys](gooeys.md) 
+* [The Towers](towers.md)
+* Components
+  * [TowerEffector](tower-effector-component.md)
+  * [TowerTargeter](tower-targeter-component.md)
+* Systems
+  * [EffectorSystem](effector-system.md)
+  * [TargeterSystem](targeter-system.md)
+* UI
+  * [ActivateScreen](activate-screen.md)
+  * [TowerScreen](tower-screen.md)
+  * [Parsers](parsers.md)
+* API
+  * [API](http://jenkins.terasology.io/teraorg/job/Terasology/job/Modules/job/G/job/GooeyDefence/job/master/javadoc/overview-summary.html)
```

Comment:
this module does not have a `master` branch, therefore this link is broken. I'm not sure what the correct link to the last "release" would be, maybe best to just remove this.

```suggestion
```